src/validators.ts

```diff
@@ -0,0 +1,46 @@
+import Joi from 'joi'
+
+export const baseRequestInputSchema = Joi.object({
+  region: Joi.string().description(
+    'The region where the property is located. If present along with address, latitude and longitude are not required.'
+  ),
+
+  address: Joi.string().description(
+    'The address of the property. If present along with region, latitude and longitude are not required.'
+  ),
+
+  latitude: Joi.number().description(
```

Comment:
I think we can add required to this too

Reply:
well, for the moment we only have support for the latitude and longitude. We remove address autocomplete for google costs